content/operate/rs/installing-upgrading/install/install-on-linux.md

```diff
@@ -29,37 +29,57 @@ To install Redis Enterprise Software, use the command line:
 
     - For Ubuntu:
-        1. Import the key:  
+        1. Import the key:
         ```sh
         gpg --import <path to GPG key>
-        ```  
-        2. Verify the package signature: 
-        ```sh 
+        ```
+        2. Verify the package signature:
+        ```sh
         dpkg-sig --verify </path-to/package.deb>
         ```
 
     - For RHEL:
-        1. Import the key:  
+        1. Import the key:
         ```sh
         rpm --import <path to GPG key>
         ```
-        2. Verify the package signature:  
+        2. Verify the package signature:
          ```sh
          rpm --checksig </path-to/package.rpm>
          ```
 
-1. To start the installation process, run the installation script. See [installation script options]({{< relref "/operate/rs/installing-upgrading/install/install-script" >}}) for a list of command-line options you can add to the following command:
+1. To start the installation process, run the installation script:
 
+    **For standard installation:**
     ```sh
     sudo ./install.sh
     ```
 
+    **For custom installation directories:**
+
+    If you need to specify custom storage paths for persistent or ephemeral storage during cluster setup, you must install Redis Enterprise Software to custom directories. See [installation script options]({{< relref "/operate/rs/installing-upgrading/install/install-script" >}}) for a complete list of command-line options.
+
+    ```sh
+    sudo ./install.sh --install-dir <path> --config-dir <path> --var-dir <path>
+    ```
+
+    For detailed information about custom installations, see [Customize installation directories]({{< relref "/operate/rs/installing-upgrading/install/customize-install-directories" >}}).
+
     {{< note >}}
-- The Redis Enterprise Software files are installed in the default [file locations]({{< relref "/operate/rs/installing-upgrading/install/plan-deployment/file-locations.md" >}}). 
+- The Redis Enterprise Software files are installed in the default [file locations]({{< relref "/operate/rs/installing-upgrading/install/plan-deployment/file-locations.md" >}}) unless you specify custom directories.
 - By default, Redis Enterprise Software runs on the OS as the `redislabs` user and `redislabs` group. If needed, you can [specify a different user and group]({{< relref "/operate/rs/installing-upgrading/install/customize-user-and-group.md" >}}) during the installation.
 - You must either be the root user or use `sudo` to run the installation script.
     {{< /note >}}
 
+    ### When to use custom installation directories
+
+    You should use custom installation directories if you need to:
+
+    - Specify custom storage paths for persistent or ephemeral storage during cluster setup
+    - Install Redis Enterprise Software in non-default locations for organizational or security requirements
+    - Avoid using root directories for the installation
+    - Meet specific compliance or infrastructure requirements
+
```

Comment:
I think this section is somewhat disruptive, especially because of the header. I think we should do one of the following:
1. Move it to https://redis.io/docs/latest/operate/rs/installing-upgrading/install/customize-install-directories/ instead. We already link to it from this install page.
2. Move it into the "For custom installation directories" intro.
3. Or both.

```suggestion
```